iwyu_test_util.py

```diff
@@ -45,6 +45,26 @@
 # // IWYU_ARGS: -Xiwyu --mapping_file=... -I .
 _IWYU_TEST_RUN_ARGS_RE = re.compile(r'^//\sIWYU_ARGS:\s(.*)$')
 
+# This is a line that specifies when a test should be skipped (because
+# the result would not be valid). IWYU_REQUIRES specifies a condition
+# that must be true for the test to execute. IWYU_UNSUPPORTED
+# specifies a condition in which the test is invalid, and would be
+# skipped. Multiple IWYU_REQUIRES and IWYU_UNSUPPORTED can be
+# specified, but they must be in a single comment block. For a test to
+# execute, all IWYU_REQUIRES conditions must be true, and all
+# IWYU_UNSUPPORTED conditions must be false.
+#
+# See the function _EvalCondition for the supported condition syntax.
+#
+# // IWYU_REQUIRES: cond
+# // IWYU_UNSUPPORTED: cond
+_IWYU_TEST_SKIP_RE = re.compile(r'^//\sIWYU_(REQUIRES|UNSUPPORTED):\s(.*)$')
+
+# List of all supported condition labels
+_IWYU_COND_LABELS=['iwyu_has_target_support_for']
+
+# All conditions have the same format 'LABEL(ARG)'
+_IWYU_COND_RE = re.compile(r'('+'|'.join(_IWYU_COND_LABELS)+')\(([A-z][A-z0-9_-]+)\)')
```

Comment:
Can you put spaces around string concatenation `+`-es, so it's easier to tell them from the regular expression?